linode/manager
# DBAAS1-106: Database Landing Table

## Preview
<img width="1524" alt="Screen Shot 2021-12-28 at 12 36 40 PM" src="https://user-images.githubusercontent.com/6440455/147592306-d47dcd7a-c643-4565-8385-adf8731bf5ef.png">


## Description

- Makes DBaaS landing table according to spec

## How to test

- Enable the MSW
- Go to `/databases` and make sure the landing table matches the proposed spec in the internal ticket description
- Check responsiveness
